vpr/src/base/stats.cpp

```diff
@@ -367,14 +367,15 @@ void print_wirelen_prob_dist() {
                 incr = index - prob_dist_size + 2;
                 prob_dist_size += incr;
 
-                float temp[prob_dist_size];
+                float* temp = new float[prob_dist_size];
```

Comment:
We should just make this a vector I think.  The code is basically doing vector.resize() operations to gradually grow an array.  If we just switch the prob_dist to a vector we can get rid of some code.

Reply:
Is there any reason why we do `if (index >= prob_dist_size) { [contents here]}` twice, or should I just leave it?